packages/pagination/src/utils/defaultGetItemAriaLabel.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  GetItemAriaLabel,
+  PaginationItemAriaLabelType,
+} from '../Pagination.types';
+
+const defaultGetItemAriaLabel: GetItemAriaLabel = ({
+  type,
+  pageNumber,
+  isSelected,
+}) => {
+  if (type === PaginationItemAriaLabelType.PageNumber) {
+    return `${isSelected ? '' : 'Go to '}page ${pageNumber}`;
```

Comment:
Is it ok to have hardcoded english in here?

Reply:
@lucasmaffazioli could you pass this "Go to" as a property?

Reply:
We changed this implementation, but we still need hardcoded strings as we have default labels. You can also check `packages/pagination/src/locale/default.ts` which is the default locale object.